.travis.yml

```diff
@@ -36,6 +36,7 @@ jobs:
       language: c
       before_install:
         - choco install python --version=3.5.4
+        - pip install jsonschema==3.2.0 types-jsonschema
```

Comment:
About pip and package version troubles: a comment in one of our Docker files says that

> Ubuntu 16.04's pip (8.1) doesn't understand the Requires-Python directive (introduced in pip 9.0), and tries to install the wrong versions of pip and setuptools.

So we could run into trouble if we use the `pip` module that comes with Ubuntu. The directive `python: 3.5` in
 `.travis.yml` says to use a different installation of Python 3.5 which comes with pip 20, so if we're getting a package that doesn't work with Python 3.5, this could be because we're accidentally invoking the system Python and its pip.